src/test/rpc/JSONRPC_test.cpp

```diff
@@ -2160,6 +2160,32 @@ class JSONRPC_test : public beast::unit_test::suite
             BEAST_EXPECT(RPC::contains_error(result));
             BEAST_EXPECT(!req[jss::tx_json].isMember(jss::Fee));
         }
+
+        {
+            // transaction with a higher base fee
+            Json::Value req;
+            test::jtx::Account const alice("alice");
+            Json::Reader().parse(
```

Comment:
Please use 
```
Json::Value
acctdelete(jtx::Account const& account, jtx::Account const& dest)
```

Reply:
I'd like to keep the norm that is used in this file's tests.

Reply:
We are moving from old norm to the new one, from direct instructions to the library functions usage

Reply:
Fixed in 00f4f81090954a980f109d95d5a566814ae76df5